.env.template

```
@@ -22,6 +22,9 @@ INCLUSION_CONNECT_BASE_URL=http://127.0.0.1:8080
 INCLUSION_CONNECT_CLIENT_ID=local_inclusion_connect
 INCLUSION_CONNECT_CLIENT_SECRET=password
 
+# parking page
+PARKING_PAGE=True
```

Je suppose que par défaut on ne veux pas la page parking donc je dirais plutôt :
```suggestion
PARKING_PAGE=False
```